employees/tests/test_unit_custom_mixins.py

```diff
@@ -99,38 +104,40 @@ def test_month_navigator_get_context_data_method_should_return_all_data_necessar
     def test_month_navigator_should_render_html_with_links_to_other_months_for_given_url(self):
         rendered_template = self._render_month_navigation_bar(2019, 1, 1)
         current_date = datetime.datetime.now()
-        self.assertTrue('<a href="/reports/project/1/2018/12/" class="btn btn-info">' in rendered_template)
+        self.assertTrue('<a href="/reports/project/1/2018/12/" class="btn btn-primary">' in rendered_template)
         self.assertTrue(
-            f'<a href="/reports/project/1/{current_date.year}/{current_date.month}/" class="btn btn-info">'
+            f'<a href="/reports/project/1/{current_date.year}/{current_date.month}/" class="btn btn-primary">'
             in rendered_template
         )
-        self.assertTrue('<a href="/reports/project/1/2019/2/" class="btn btn-info">' in rendered_template)
+        self.assertTrue('<a href="/reports/project/1/2019/2/" class="btn btn-primary">' in rendered_template)
 
     def test_month_navigator_should_not_render_html_with_link_to_next_month_if_upper_limit_is_met(self):
         rendered_template = self._render_month_navigation_bar(2099, 12, 1)
         current_date = datetime.datetime.now()
-        self.assertTrue('<a href="/reports/project/1/2099/11/" class="btn btn-info">' in rendered_template)
+        self.assertTrue('<a href="/reports/project/1/2099/11/" class="btn btn-primary">' in rendered_template)
         self.assertTrue(
-            f'<a href="/reports/project/1/{current_date.year}/{current_date.month}/" class="btn btn-info">'
+            f'<a href="/reports/project/1/{current_date.year}/{current_date.month}/" class="btn btn-primary">'
             in rendered_template
         )
-        self.assertFalse('<a href="/reports/project/1/2100/1/" class="btn btn-info">' in rendered_template)
+        self.assertFalse('<a href="/reports/project/1/2100/1/" class="btn btn-primary">' in rendered_template)
 
     def test_month_navigator_should_not_render_html_with_link_to_previous_month_if_lower_limit_is_met(self):
         rendered_template = self._render_month_navigation_bar(2019, 5, 1)
         current_date = datetime.datetime.now()
-        self.assertTrue('<a href="/reports/project/1/2019/6/" class="btn btn-info">' in rendered_template)
+        self.assertTrue('<a href="/reports/project/1/2019/6/" class="btn btn-primary">' in rendered_template)
         self.assertTrue(
-            f'<a href="/reports/project/1/{current_date.year}/{current_date.month}/" class="btn btn-info">'
+            f'<a href="/reports/project/1/{current_date.year}/{current_date.month}/" class="btn btn-primary">'
             in rendered_template
         )
-        self.assertFalse('<a href="/reports/project/1/2000/4/" class="btn btn-info">' in rendered_template)
+        self.assertFalse('<a href="/reports/project/1/2000/4/" class="btn btn-primary">' in rendered_template)
```

Comment:
Sorry for that. 😅 At the time I wanted to make sure that the page contains the link, so I copied the whole tag. In that case, maybe you could just dump this whole class attribute and just leave it as `'<a href="/reports/project/1/2000/4/"'`. It won't be a problem next time we change the button. And it's not at all crucial for this test. 😝

Reply:
Done